kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -744,6 +831,11 @@ define([
 
     const BatchMixin = (Base) =>
         class extends Base {
+            constructor(config) {
+                // run the constructor for the base class
+                super(config);
+                this.pollInterval = config.pollInterval || DEFAULT_STATUS_REQUEST_INTERVAL;
```

Comment:
ensure that the poll interval for looping batch job status requests is configurable